fronts-client/src/actions/Collections.ts

```diff
@@ -68,8 +67,10 @@ import { fetchCollectionsStrategy } from 'strategies/fetch-collection';
 import { updateCollectionStrategy } from 'strategies/update-collection';
 import { getPageViewDataForCollection } from 'actions/PageViewData';
 import { isMode } from 'selectors/pathSelectors';
+import { groupBy, uniqBy } from 'lodash';
+import { fetchChefsById } from 'bundles/chefsBundle';
 
-const articlesInCollection = createSelectAllArticlesInCollection();
+const articlesInCollection = createSelectAllCardsInCollection();
```

Comment:
Presumably `articlesInCollection` could also be renamed `cardsInCollection` (but not essential, obvs)

Reply:
Yep, missed this, thanks for spotting! e91b656f981d50d9730aaf169ba88eec5314769e